ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5090,3 +5090,22 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: userExistenceApi
+    request_path: "{{ user_service_prefix  }}/v1/exists"
+    upstream_url: "{{ learning_service_url }}/v1/user/exists"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: "{{ getUserByKey_ACL | default(['appUpdate']) }}"
```

Comment:
Give the same name as the API for the config.whitelist instead of getUserByKey_ACL give userExistenceApi_ACL